app/helpers/results_document_helper.rb

```diff
@@ -1,11 +1,16 @@
 module ResultsDocumentHelper
 
-  def get_main_title document
+  def get_main_title(document)
     (document['title_display'] || "").html_safe
   end
 
+  def get_main_title_date(document)
+    # MODS data is indexed with the display date in Solr field pub_year_ss
+    date = document["pub_year_ss"].to_s
+    return "[#{date}]" unless date.empty?
```

Comment:
Instead of casting `document["pub_year_ss"]` to a string and then checking if the string is `empty?` we can simply just check if `date` is `present?` (and drop the `to_s`).

This avoids the needless string casting which might be useful given that this code is executed a minimum of 20 times for every search result.